lib/push-manager.js

```diff
@@ -248,9 +248,10 @@ PushManager.prototype.notifyById = function (installationId, notification, cb) {
 PushManager.prototype.notifyByQuery = function(installationQuery, notification, cb) {
   assert.ok(cb, 'callback should be defined');
   var self = this;
-  var filter = { where: installationQuery };
+  var filter = installationQuery;
```

Comment:
Yes, I am aware of it, but as it is said in the issue, it does not work that way. Only the where is needed here to use it with the `inq` operator.
A sample code would look like the following:
```
var PushModel = app.models.Push;

User.notify = function(cb, userIds, msgNote) {
		
		var Notification = app.models.Notification;
	
		var note = new Notification({
			message: msgNote || 'default message'
		});

		PushModel.notifyByQuery({userId: {inq: userIds}}, note, function(err) {
			if (err) return cb(err);
			debug('Pushing notification to ' + userIds + ' with message: ', note.message);
			cb(null);
		});

	};
```
This code (as it is in the [documentation](https://docs.strongloop.com/display/public/LB/Push+notifications#Pushnotifications-Sendpushnotifications)) should work, but it does not without my modification.
When I debugged it, found that the `installationList` will be empty because of the incorrect filter, which led to this PR.